doc/source/how-to-run-flower-using-docker.rst

```diff
@@ -172,20 +175,36 @@ The ``Dockerfile`` contains the instructions that assemble the SuperNode image.
 
 In the first two lines, we instruct Docker to use the SuperNode image tagged ``nightly`` as a base
 image and set our working directory to ``/app``. The following instructions will now be
-executed in the ``/app`` directory. Next, we install the ``ClientApp`` dependencies by copying the
+executed in the ``/app`` directory. Next, we install the ClientApp dependencies by copying the
 ``requirements.txt`` file into the image and run ``pip install``. In the last two lines,
-we copy the ``ClientApp`` code (``client.py`` and ``task.py``) into the image and set the entry
+we copy the ClientApp code (``client.py`` and ``task.py``) into the image and set the entry
 point to ``flower-client-app``.
 
+.. important::
+
+  If the requirement.txt contains the `flwr <https://pypi.org/project/flwr/>`__ or
+  `flwr-nightly <https://pypi.org/project/flwr-nightly/>`_ package, please ensure the version in
+  requirement.txt matches the docker image version.
```

Comment:
I think the way you phrased it sounds better. I will go with that. The reason I included the note is because I'm afraid  users develop locally with a requirements.txt containing the flwr package and once they're done they just copy the same file into the Docker image.

Reply:
I can see why that can be the case. Since this is a self-contained "How-to" page, I think it's important to start from the simplest setup. This basic Dockerfile can then be expanded by users for other more complicated server setups.